src/libultra/os/settimer.c

```diff
@@ -21,9 +21,7 @@ int osSetTimer(OSTimer* t, OSTime value, OSTime interval, OSMesgQueue* mq, OSMes
     saveMask = __osDisableInt();
 
     if (__osTimerList->next != __osTimerList) {
-        if (0) {
-            ;
-        }
+        if (1) {}
```

Comment:
Weren't we adding `//! FAKE` to those?

Reply:
I can, but there are a lot in the code base without.

Reply:
I was thinking on maybe you could add the FAKE to the ifs you were cleaning up, but it is fine either way